.github/workflows/update-and-publish-helm-chart.yml

```diff
@@ -1,16 +1,21 @@
 name: update-and-publish-cromwell-helm-chart
 on:
-  # push:
-  #   branches:
-  #     - 'main'
-  workflow_dispatch:
+  push:
+    branches:
+      - 'main'
 
 jobs:
   update-and-publish:
     if: ( !startsWith(github.event.head_commit.message, 'Auto update cromwell-helm chart version') )
     name: Update and publish new cromwell-helm chart
     runs-on: ubuntu-latest
     steps:
+      - name: Ensure that a Jira ID is present in the commit message
+        run: |
+          JIRA_ID=$(echo '${{ github.event.head_commit.message }}' | grep -Eo '\[?[A-Z][A-Z]+-[0-9]+\]?')
```

Comment:
Some thoughts:
- Does this work on multi-line messages?
- Does this work if the jira ID is embedded in `[]`s? Or in other ways not at the start of the message?
  - If we eventually add a "Does the title contain a jira ID" check, we should duplicate the same pattern matching we use here to pick it out of the title.
- Do we want an allow-list of JIRA project codes we might use (to reduce the chance of falsely identifying a random part of the message string as a jira ID)?
- This is a great candidate for "if it fails, alert the team in slack"

Reply:
> Does this work on multi-line messages?
> Does this work if the jira ID is embedded in []s? Or in other ways not at the start of the message?

I believe: yes, yes, and yes, based on some hand tests that I did

> Do we want an allow-list of JIRA project codes we might use (to reduce the chance of falsely identifying a random part of the message string as a jira ID)?

Maybe? I'm on the fence... let's talk about this tomorrow

> This is a great candidate for "if it fails, alert the team in slack"

Agreed. I'll add it to that ticket